docs/docs/faq/question_05.md

```diff
@@ -0,0 +1,46 @@
+
+# How should I save and load objects in `sbi`?
+
+`NeuralPosterior` objects are picklable.
+```python
+import pickle
+
+# ... run inference
+posterior = inference.build_posterior()
```

Comment:
```suggestion
from sbi.inference import SNPE, prepare_for_sbi, simulate_for_sbi

simulator, prior = prepare_for_sbi(simulator, prior)
inference = SNPE(prior)

theta, x = simulate_for_sbi(simulator, proposal=prior, num_simulations=1000)
density_estimator = inference.append_simulations(theta, x).train()
posterior = inference.build_posterior(density_estimator)
```

Even though, it might appear overhead, it is nice to encounter something known from the tutorials for a novice.